keepassxc-browser/background/client.js

```diff
@@ -326,6 +339,8 @@ function onDisconnected() {
 }
 
 keepassClient.onNativeMessage = function(response) {
+    console.log(JSON.stringify(response));
```

Comment:
Did you add this for debugging or perpetual?

Reply:
This was a mistake. I was just using it for debugging some other stuff. Sigh.